internal/build/pom.xml

```diff
@@ -1054,5 +1057,14 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <!-- Profile to skip time-consuming steps. -->
+            <id>fast</id>
+            <properties>
+                <basepom.check.skip-all>true</basepom.check.skip-all>
```

Comment:
sorry, I was not clear in my review. the `<skipTests>true</skipTests` can stay just fine. What I don't want us is to not compile the classes. Not running the tests is ok.

Reply:
glad to read that

done